app/src/main/java/com/udacity/nanodegree/blooddonation/util/AppUtil.java

```diff
@@ -14,4 +18,30 @@ public static void replaceFragmentInActivity(FragmentManager fragmentManager, Fr
             int containerId) {
         fragmentManager.beginTransaction().replace(containerId, fragment).commit();
     }
+
+    public static void validatePhoneNumber(EditText phoneNumberView , View iAmInButtonView){
```

Comment:
Please, don't have listener here, let this function just accept, count and text and make this function return true or false.

Also, move the logic in Util class.